app/test/frontend/golden/pkg_index_page.html

```diff
@@ -179,6 +179,7 @@ <h3 class="foldable-button">Dart
     packages
 
 
+
```

Comment:
Is this extra newline intentional?

Reply:
That is coming from the the mustache template. We could optimize for nicer outputs but then the source templates would be less readable.

Reply:
And yes, it is expected from the template change (but not intentional).